CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 
 ## Bug fixes
 
+-   Removed class `Simplification` ([#1369](https://github.com/pybamm-team/PyBaMM/pull/1369))
```

Comment:
```suggestion
-   Removed `Simplification` class and `.simplify()` function ([#1369](https://github.com/pybamm-team/PyBaMM/pull/1369))
```
and this should go under "breaking changes" instead

Reply:
Okay 👍